converter.go

```diff
@@ -113,12 +113,7 @@ func valueToString(v driver.Value, tsmode string) (*string, error) {
 				s := fmt.Sprintf("%d",
 					(tm.Hour()*3600+tm.Minute()*60+tm.Second())*1e9+tm.Nanosecond())
 				return &s, nil
-			case "TIMESTAMP_NTZ":
```

Comment:
This behavior is TBD. 
I think the change for TIMESTAMP_TZ is correct.
Overall the correct behavior would be:
TIMESTAMP_NTZ would use the wall clock time without timezone info in time.Time.
TIMESTAMP_TZ would use the epoch time in UTC.

The correct behavior should just be the opposite of old behavior.

Reply:
I think TIMESTAMP_NTZ behavior is ok, since the value is stored in Time, which is monotonic clocks:
https://golang.org/pkg/time/

Changing to wall clock should be done by the application.